src/common/components/Service/AddService.vue

```diff
@@ -246,8 +246,7 @@ export default {
       ],
       totalPrice: [
         rulesHandler.FIELD_REQUIRED,
-        val => /^[0-9]+$/.test(val) || "Service price is invalid",
-        rulesHandler.MAX_CHARACTER(4)
+        val => /^\d+(\.\d{1,4})?$/.test(val) || "Service price is invalid"
```

Comment:
why do you limit 4 digits after the comma?

Reply:
it's requirement from QA

Reply:
as I know, you should ensure that a number is absolute or decimal. Not a digit constraint, as it will be constrained when displayed in ui by the Front end using `toFixed`

cc @hilyds

Reply:
validation is still possible to fill in absolute numbers, but when filling in decimal digits the digits after the comma will be limited